src/components/ProfilePage/LevelBox/PointsBox.tsx

```diff
@@ -1,18 +1,31 @@
 import { Box } from '@mui/material';
 import { IconHeader } from '../Common/IconHeader';
-import { CenteredBox } from '@/components/ProfilePage/ProfilePage.style';
 import { PointsDisplay } from './PointsDisplay';
+import { XPIcon } from 'src/components/illustrations/XPIcon';
+import { XPIconHeader } from 'src/components/illustrations/IconXP';
 interface PointsBoxProps {
   points?: number;
 }
 
 export const PointsBox = ({ points }: PointsBoxProps) => {
+  const currentDate = new Date();
+  const options: Intl.DateTimeFormatOptions = {
```

Comment:
Would be good to use a translation (one already exists for that purpose if I recall correctly or you can reuse what exists for currency/decimal as example)